cl/users/tasks.py

```diff
@@ -98,3 +99,35 @@ def send_failed_email(
             raise self.retry(exc=exc)
         failed_email.status = STATUS_TYPES.SUCCESSFUL
         failed_email.save()
+
+
+@app.task
+def check_recipient_deliverability(
+    recipient: str,
+    backoff_prev_counter: int,
+) -> None:
+    """This task checks if the recipient's email address is deliverable, it
+    works by verifying if the backoff event retry counter was updated since the
+    task was scheduled if so it means that it came in a new bounce event for
+    the recipient, otherwise it means that the recipient is deliverable,
+    so the backoff event is deleted and the waiting failed emails are scheduled
+    to be sent.
```

Comment:
A big difference between English and Spanish that I learned when I studied Spanish is that in English, sentences like the above are considered "run-on sentences", while in Spanish they're very normal (and even considered *good*). 

It's a funny thing, but I think it'll help your code/issues/PRs/etc if you try to add a lot more periods to your work. So:

```suggestion
    """This task checks if the recipient's email address is deliverable. It
    works by verifying if the backoff event retry counter was updated since the
    task was scheduled. If so, it means that it came in a new bounce event for
    the recipient. Otherwise, it means that the recipient is deliverable,
    so the backoff event is deleted and the waiting failed emails are scheduled
    to be sent.
```

I've hesitated to mention this because I'm not here to help with English, but it'll make your writing a lot easier to understand. I did a little digging and thought [this was a good reference](https://www.espanolavanzado.com/gramatica-avanzada/1675-comma-splice-in-spanish-oraciones-yuxtapuestas) (with links to Spanish ones).

Reply:
Thank you! I appreciate this feedback since it help me to improve my documentation, I applied these changes.